google/dev/delete_resources.py

```diff
@@ -0,0 +1,399 @@
+#!/usr/bin/python
+#
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=import-error
+# pylint: disable=star-args
+"""
+Implements program for deleting Google API resources by their name and/or age.
+
+Example usage:
+  # Delete all images in my project more than 30 days old.
+  python delete_resources.py \
+      compute.images \
+      --project=myproject \
+      --age=30 \
+      --dry_run
+
+  # Delete all the versions of all the objects in gs://mybucket
+  python delete_resources.py \
+      storage.objects \
+      --delete_kwargs=bucket=mybucket \
+      --list_kwargs=versions=true \
+      --dry_run
+
+  # Delete all the instances from this project that have 'xyz' in the name.
+  # Use the list_aggregated method, which considers all zones.
+  python delete_resources.py \
+      compute.instances \
```

Comment:
Are the options for resources enumerated anywhere? A link to the docs or just a list here would suffice.